core/runtime/pom.xml

```diff
@@ -60,6 +60,11 @@
             <artifactId>artemis-core-client</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Can we add a comment explicitly stating that we need this dependency for `artemis-core-client`?

I highly suspect that we need this dpenendency in [`jms/runtime/pom.xml`](https://github.com/quarkiverse/quarkus-artemis/blob/0d9a2ab0751707d28a0ac41948b870b733f46fce/jms/runtime/pom.xml) as well.

Reply:
Okay, the dependency is not needed in [`jms/runtime/pom.xml`](https://github.com/quarkiverse/quarkus-artemis/blob/0d9a2ab0751707d28a0ac41948b870b733f46fce/jms/runtime/pom.xml); the `jms/runtime` depends on the `core/runtime`, so this change is "propagated".

Reply:
Nope :slightly_smiling_face:

Reply:
@zhfeng did you forget to add the comment?